### ISSUE TITLE
Brand Audit - Enhance plaque test

### ISSUE THREAD

**Issue by halocline** [grommet/hpe-design-system#3154]
Enhance plaque test by applying a more granular approach to issue highlighting.

**Deliverables**
- [x] #3155
- [x] Create distinct highlights for specific issues. 
- [x] Combine highlight and description in some sort of legend with key.
- [x] Submit and merge PR into AUDIT branch. https://github.com/grommet/grommet-theme-hpe/pull/324
- [x] Follow on: https://github.com/grommet/hpe-design-system/issues/3173

**Comment by halocline** [comment]
Staus, Feb 24 -
- Audit branch and release of audit-stable | COMPLETE
- Creating highlights for distinct issues | IN PROGRESS
  - Generated "wishlist" of issues to identify
  - Writing and testing issue rules. Likely will only complete a subset of wishlist by end of sprint, but will be a significant improvement in granularity and specificity from v1 plaque test.

Plan for next week:
- Submit PR on Monday
- Communicate with Sam once PR is merged. Targeting Tuesday midday.

**Comment by halocline** [comment]
Initial PR merged into AUDIT branch and follow on issue created.

**Comment by halocline** [comment]
Initial PR merged into AUDIT branch and follow on issue created.
